crates/klave/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "klave"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
Can you please not more the version number as we discussed last time, this is done by the release runner. I can't force push branches on your repo, so you'll have to do it 😉